pyomo/dae/initialization.py

```diff
@@ -87,7 +87,8 @@ def get_inconsistent_initial_conditions(model, time, tol=1e-8, t0=None,
     return list(inconsistent)
 
 
-def solve_consistent_initial_conditions(model, time, solver, tee=False):
+def solve_consistent_initial_conditions(model, time, solver, tee=False,
+        allow_skip=True, suppress_warnings=False):
```

Comment:
@Robbybp could you expand the docstring with descriptions of these new options? It isn't really clear to me what the `allow_skip` option is doing by the name alone.